Automattic/wc-calypso-bridge
# Remove appearance task and handle unregister plugin JS

### Changes proposed in this Pull Request:

DO NOT MERGE until https://github.com/Automattic/wc-calypso-bridge/pull/1357 is merged and Jan 2, 8pm PT

This PR:

- Removes appearance task
- Fixes cases where JS plugins are registered earlier than `plugins.pluginRegistered` action hook





### How to test the changes in this Pull Request:

1. If https://github.com/Automattic/wc-calypso-bridge/pull/1357 is not yet merged, enable CYS feature flag:
    1. Open `_cli` on your Woo Express or free trial site, example `https://yoursite.wpcomstaging.com/_cli`
    2. Run `plugin install https://gist.github.com/ilyasfoo/6c5900753edcf06faef9a4b04dca23a8/archive/0c3e3f4d3de60b84fe7e8b950ffcf753f7ec1ddf.zip --activate`
2. Go to Homescreen
3. Observe `Customize your store` task is loaded
4. Observe other tasks are also loaded properly
5. Click on `Start Customizing`
6. Observe you're redirected to Customize Store screen



### Other information:




### FOR PR REVIEWER ONLY:
